_pages/home.md

```diff
@@ -25,7 +26,7 @@ This is an open call to all RSEs and anyone involved with research software worl
 
 Any questions, read [more](faq/about/what-is-sorse) or [get in touch](contact/)!
```

Comment:
The end of this paragraph is sort of the first "call to action" so we might want to draw the reader to upcoming events - saying like "Here are events happening soon that you can participate in!"
```suggestion
Any questions, read [more](faq/about/what-is-sorse), [get in touch](contact/), or see [upcoming events](#upcoming-events)!
```